test/routes/vary.test.ts

```diff
@@ -1,4 +1,3 @@
-import { mocked } from 'ts-jest/utils'
```

Comment:
`ts-jest/utils` now spews out a load of warnings saying that its deprecated and to move over to a new version of jest. 

This just isn't needed though and it's generally cleaner to not used `mocked` at all. 

The rest of the changes in this PR are to remove usage of `mocked`